assets/javascripts/discourse/components/new-resolution-button.js.es6

```diff
@@ -4,6 +4,6 @@ import Component from "@ember/component";
 export default Component.extend({
   actions: {
     clickCreateResolutionButton() {
-      showModal("resolution-ui-builder");
+      showModal("resolution-ui-builder")._setupPoll();
```

Comment:
I think that we don't need to call `_setupPoll()` since it will be automatically called in `init()` function

Reply:
I suppose `init()` isn't being called every time, because when I open edit resolution modal and close it without saving, then open new resolution modal, it is filled with resolution I tried to edit.